### ISSUE TITLE
Support not replacing variables when not found

### ISSUE THREAD

**Issue by headstar** [bhirsz/robotframework-templateddata#3]
Would be useful to be able to leave variables untouched if not found. 

The variable might not be a Robot variable, e.g:
`{
  "foo" : "${FOO_ROBOT_VARIABLE}"
  "bar" : "${json-unit.any-string}"
}`

**Comment by bhirsz** [comment]
Thank you for opening issue ;) I must admit it's rather small library that I have created some time ago when I needed it to refactor some of our tests that used unimaginable amount of duplicates instead of using templates. That being said if there are actually people using it I can support it.

Proposed change makes sense and it could be easily done. It could be controlled via flag, ie:
```
Library    TemplatedData  ignore_empty=True
```

`ignore_empty` may not be the best name but it's matches already existing `default_empty`. If you have better ideas (for naming) - let me know.

And feel free to open other feature requests when necessary ;)

**Comment by bhirsz** [comment]
For now went with `ignore_missing`. The only problem is that if you set it to True it will ignore any default values, ie:
```
${MISSING_BUT_WITH_DEFAULT:5}
```
Is this fine for you? Or it should be improved (for example ignore only those that don't have default value set).

**Comment by headstar** [comment]
Thanks for not resolving this with "Won't Do" :)

"The only problem is that if you set it to True it will ignore any default values"

Yeah, that's fine with me (hard to avoid that downside I think).

**Comment by bhirsz** [comment]
Linked PR - when it will be merged I will release it as 1.2.0 (let me known if there are any issues in new version).

Example template before:
```
{
  "accounts": [
    {
      "id": "${ROBOT_VARIABLE:5}"
    },
    {
      "id2": "${NOT_ROBOT_VARIABLE}"
    },
    {
      "id3": "${NOT_ROBOT_VARIABLE:5}"
    },
    {
      "foo": "${ROBOT_VARIABLE}",
      "bar": "${json-unit.any-string}"
    },
    {
      "test": "@{LIST_BUT_NOT_ROBOT_VARIABLE}"
    }
  ]
} 
```
with `ignore_missing` flag set to True:
```
{
  "accounts": [
    {
      "id": "VALUE"
    },
    {
      "id2": "${NOT_ROBOT_VARIABLE}"
    },
    {
      "id3": "${NOT_ROBOT_VARIABLE:5}"
    },
    {
      "foo": "VALUE",
      "bar": "${json-unit.any-string}"
    },
    {
      "test": "@{LIST_BUT_NOT_ROBOT_VARIABLE}"
    }
  ]
} 
```
